dev.zed.Zed.yaml

```diff
@@ -51,6 +51,56 @@ modules:
         sha256: bad10f3c553a0e1854649ab59c5b2434da22ca1a54ae6138f1f53961567e1ab7
         url: https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 
+  - name: libmd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libmd-1.1.0.tar.xz
+        sha256: 1bd6aa42275313af3141c7cf2e5b964e8b1fd488025caf2f971f43b00776b332
+        x-checker-data:
+          type: anitya
+          project-id: 15525
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libmd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
+
+  - name: libbsd
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://libbsd.freedesktop.org/releases/libbsd-0.12.2.tar.xz
+        sha256: b88cc9163d0c652aaf39a99991d974ddba1c3a9711db8f1b5838af2a14731014
+        x-checker-data:
+          type: anitya
+          project-id: 1567
+          stable-only: true
+          url-template: https://libbsd.freedesktop.org/releases/libbsd-$version.tar.xz
+    cleanup:
+      - /lib/pkgconfig
+      - /share/man
+
+  - name: netcat
+    buildsystem: simple
+    build-commands:
+      - |
+        while read patch; do
+          echo "Applying $patch..."
+          patch -Np1 -i "debian/patches/$patch"
+        done < debian/patches/series
+      - make CFLAGS="$CFLAGS -I/usr/include/libbsd" LDFLAGS="$LDFLAGS -lbsd"
+      - install -Dm0755 nc /app/bin/nc
+    sources:
+      - type: archive
+        url: https://salsa.debian.org/debian/netcat-openbsd/-/archive/debian/1.228-1/netcat-openbsd-debian-1.228-1.tar.gz
```

Comment:
Why the upstream tarball is not used here?

Reply:
The upstream tarball is for BSD only and doesn't work on Linux without patches, this Debian package bundles those patches, is tracked by Anitya, and is also used by both [Arch Linux](https://gitlab.archlinux.org/archlinux/packaging/packages/openbsd-netcat/-/blob/main/PKGBUILD?ref_type=heads) and [Alpine Linux](https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/main/netcat-openbsd/APKBUILD).